dd-sdk-android/src/main/kotlin/com/datadog/android/rum/GlobalRum.kt

```diff
@@ -107,6 +112,15 @@ object GlobalRum {
 
     // region Internal
 
+    internal fun addUserInteraction() {
```

Comment:
do you still need this as you are doing everything in the `updateSessionIdIfNeeded` ?

Reply:
This is needed to update the `lastInteraction` time. `updateSessionIdIfNeeded` only updates it when it generates a new Session, this method allows to extend the duration of the current session